playwright.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from "@playwright/test";
+import dotenv from "dotenv";
+import path from "path";
+
+// Read from default ".env" file.
+dotenv.config();
+
+// Alternatively, read from "../my.env" file.
+dotenv.config({ path: path.resolve(__dirname, "..", "my.env") });
+
+export default defineConfig({
+  use: {},

```

Comment:
I didn't do my research too deeply on what this `defineConfig` call does and if it needs to be exported or not. You could try removing it to see if it still adds your variables in. If they stop working then add it back 😅